src/SourceBuild/tarball/content/src/source-mappings.json

```diff
@@ -0,0 +1,152 @@
+            "defaultRemote": "https://github.com/dotnet/aspnetcore",
+            "exclude": [
+                "**/samples/**/jquery-validation-unobtrusive/.bower.json",
+                "**/samples/**/jquery-validation-unobtrusive/*.js"
+            ]
+        },
+        {
+            "name": "clicommandlineparser",
+            "defaultRemote": "https://github.com/dotnet/clicommandlineparser"
+        },
+        {
+            "name": "command-line-api",
+            "defaultRemote": "https://github.com/dotnet/command-line-api"
+        },
+        {
+            "name": "deployment-tools",
+            "defaultRemote": "https://github.com/dotnet/deployment-tools"
+        },
+        {
+            "name": "diagnostics",
+            "defaultRemote": "https://github.com/dotnet/diagnostics"
+        },
+        {
+            "name": "format",
+            "defaultRemote": "https://github.com/dotnet/format"
+        },
+        {
+            "name": "fsharp",
+            "defaultRemote": "https://github.com/dotnet/fsharp"
+        },
+        {
+            "name": "installer",
+            "defaultRemote": "https://github.com/dotnet/installer",
+            "defaultRef": "vmr"
+        },
+        {
+            "name": "linker",
+            "defaultRemote": "https://github.com/dotnet/linker"
+        },
+        {
+            "name": "msbuild",
+            "defaultRemote": "https://github.com/dotnet/msbuild"
+        },
+        {
+            "name": "nuget-client",
+            "defaultRemote": "https://github.com/NuGet/NuGet.Client",
+            "defaultRef": "dev",
+            "exclude": [
+                "**/EndToEnd/**/jquery-validation-unobtrusive/*.js",
+                "**/EndToEnd/**/jquery-validation-unobtrusive/.bower.json"
+            ]
+        },
+        {
+            "name": "razor-compiler",
+            "defaultRemote": "https://github.com/dotnet/razor-compiler"
+        },
+        {
+            "name": "roslyn",
+            "defaultRemote": "https://github.com/dotnet/roslyn"
+        },
+        {
+            "name": "roslyn-analyzers",
+            "defaultRemote": "https://github.com/dotnet/roslyn-analyzers"
+        },
+        {
+            "name": "runtime",
+            "defaultRemote": "https://github.com/dotnet/runtime"
+        },
+        {
+            "name": "sdk",
+            "defaultRemote": "https://github.com/dotnet/sdk"
+        },
+        {
+            "name": "source-build",
+            "defaultRemote": "https://github.com/dotnet/source-build"
+        },
+        {
+            "name": "source-build-externals",
+            "defaultRemote": "https://github.com/dotnet/source-build-externals",
+            "exclude": [
+                "**/humanizer/samples/**/*.js"
+            ]
+        },
+        {
+            "name": "source-build-reference-packages",
+            "defaultRemote": "https://github.com/dotnet/source-build-reference-packages",
+            "exclude": [
```

Comment:
If we are going to want to move source-build over to the VMR, these are going to be required until the repos remove their full fx tfm dependencies.